index.html

```diff
@@ -1,21 +1,78 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
-    <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
-    <link rel="stylesheet" href="css/normalize.css">
-    <!-- Add a link to your CSS file here (use the line above to guide you) -->
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <title>Karma</title>
+  <link href="//fonts.googleapis.com/css?family=Roboto:400,500,300" rel="stylesheet" type="text/css" />
+  <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet" />
+  <link rel="stylesheet" href="css/normalize.css" />
+  <link rel="stylesheet" href="css/style.css" />
+  <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
+
 <body>
+  <header id="header">
+    <nav class="nav-bar">
+      <img class="logo" src="img/karma-logo.svg" alt="karma-logo" />
+      <ul>
+        <li class="bold"><a href="">Meet Karma</a></li>
+        <li><a href="">How it Works</a></li>
+        <li><a href="">Store</a></li>
+        <li><a href="">Blog</a></li>
+        <li><a href="">Help</a></li>
+        <li><a href="">Login</a></li>
+      </ul>
+    </nav>
+  </header>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+  <main>
+    <section id="hero">
+      <h1>Introducing Karma</h1>
+      <h3>Bring WiFi with you, everywhere you go.</h3>
+      <button type="button" class="button"> <a href="#">Learn More</a>
+    </section>
+    <section id="benefits">
+      <h2>Everyone needs a little Karma.</h2>
+      <ul class="icons">
+        <li>
+          <figure>
+            <img class="icon" src="img/icon-devices.svg" alt="icondevices" />
+            <figcaption>Internet for all devices</figcaption>
+          </figure>
+        </li>
+        <li>
+          <figure>
+            <img class="icon" src="img/icon-coffee.svg" alt="icon-coffee" />
+            <figcaption>Boost your productivity</figcaption>
+          </figure>
+        </li>
+        <li>
+          <figure>
+            <img class="icon" src="img/icon-refill.svg" alt="icon-refill" />
+            <figcaption>Pay as You Go</figcaption>
+          </figure>
+        </li>
+      </ul>
+    </section>
+  </main>
 
+  <footer>
+    <div>
```

Comment:
Good structure for the footer, except for this `div` - what does it do? Is it necessary?